hack/build-base-images.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# This script builds the base and release images for use by the release build and image builds.
```

Comment:
we're not actually going to do this in this repo are we?  I'd expect base+release images to still come out of origin.

Reply:
I'm going to remove it for now but we'll probably end up needing a way for this to be reused.  The CI system needs to reuse the latest valid origin-base, even if it's named something else.  Steve and I will have to talk about it.